packages/apollo/src/commands/client/extract.ts

```diff
@@ -9,7 +10,13 @@ import { ClientIdentity } from "apollo-language-server";
 export default class ClientExtract extends ClientCommand {
   static description = "Extract queries from a client";
   static flags = {
-    ...ClientCommand.flags
+    ...ClientCommand.flags,
+    stripLiterals: flags.boolean({
```

Comment:
Since "literals" also applies to object and list literals (which are not stripped), perhaps we should be a bit more clear with this option name.  Even if it's a bit wordy, the intent is more clear, I think?  (And its less necessary for someone to understand the notion of negated flags and perhaps more intrinsic understanding of the default behavior?).  This change would require changing the `default` to `false` below and flipping other conditionals too, of course.)

```suggestion
    preserveStringAndNumericLiterals: flags.boolean({
```